meta/main.yml

```diff
@@ -8,4 +8,5 @@ galaxy_info:
   categories:
   - firewall
   - system
-dependencies: []
+dependencies:
+  - { role: 'repo-epel', when: ansible_os_family|lower == "redhat" }
```

Comment:
In redhat os family ferm is in yum epel repository. I use this role https://github.com/geerlingguy/ansible-role-repo-epel (install via ansible-galaxy on my own directory and create symbolink name repo-epel in roles directory) to install yum epel repository.